distributions/docker/solr/pom.xml

```diff
@@ -4,7 +4,7 @@
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <parent>
     <artifactId>docker</artifactId>
-    <groupId>replication.distributions</groupId>
+    <groupId>com.connexta.replication.distributions</groupId>
```

Comment:
❓ Can we switch these 2 lines to always define the group id followed by the artifact id?

Reply:
Changed